modules/representation_management/spec/requests/v0/accredited_entities_for_appoint_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'RepresentationManagement::V0::AccreditedEntitiesForAppointController', type: :request do
+  let(:path) { '/representation_management/v0/accredited_entities_for_appoint' }
+
+  before do
+    Flipper.enable(:appoint_a_representative_enable_pdf)
+  end
+
+  context 'the response should be an empty array' do
+    before do
+      create(:accredited_individual, :with_location, full_name: 'Bob Law')
+      create(:accredited_individual, :with_location, full_name: 'Bob Smith')
+      create(:accredited_organization, :with_location, name: 'Bob Law Firm')
+      create(:accredited_organization, :with_location, name: 'Bob Smith Firm')
+    end
+
+    context 'when the query parameter is an empty string' do
+      it 'returns an empty array' do
+        get path, params: { query: '' }
+
+        parsed_response = JSON.parse(response.body)
+        expect(parsed_response).to eq([])
+      end
+    end
+
+    context 'when the query parameter is not present' do
+      it 'returns an empty array' do
+        get path
+
+        parsed_response = JSON.parse(response.body)
+        expect(parsed_response).to eq([])
+      end
+    end
+
+    it 'when there are no matching results' do
+      AccreditedIndividual.destroy_all
+      AccreditedOrganization.destroy_all
```

Comment:
I think this test is more powerful without the destroy and it or a copy of it should be in the model to test the word similarity functionality

Reply:
Good catch, this was leftover from an iteration where we were using all Levenshtein distance, so all results up to 10 would be returned.